packages/scaffold/src/partials/w3m-account-wallet-features-widget/index.ts

```diff
@@ -97,10 +97,11 @@ export class W3mAccountWalletFeaturesWidget extends LitElement {
       <wui-profile-button
         @click=${this.onProfileButtonClick.bind(this)}
         address=${ifDefined(this.address)}
+        profileName=${ifDefined(this.profileName)}
         networkSrc=${ifDefined(networkImage)}
         icon="chevronBottom"
         avatarSrc=${ifDefined(this.profileImage ? this.profileImage : undefined)}
-        ?isprofilename=${Boolean(this.profileName)}
+        ?isProfileName=${Boolean(this.profileName)}
```

Comment:
This should be changed in https://github.com/WalletConnect/web3modal/pull/2276/files, `isProfileName` shouldn't be needed